PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing
+
+## Pull Request Etiquette
+
+There are several guidelines you should follow in order for your
+  Pull Request to be merged.
+
+- [ ] I have checked the PRs for upcoming features/bug fixes.
+- [ ] I have contacted the staff regarding bigger changes.
+- [ ] I have read the [contributing][contributing] rules.
+- [ ] I have updated my fork to the latest `upstream/master` changes.
+- [ ] I have provided proper commit captions.
+- [ ] I have included documentation if applicable.
+- [ ] I have made my code consistent with the rest of JDA.
+
```

Comment:
`I have contacted the staff regarding bigger changes` should not be here
Everything that is already mentioned in the contributing guidelines can be left out here.
`contribution guidelines` rather than `contributing rules`